### PR TITLE
Add - about FAQs pattern

### LINKED ISSUE #67: Block Patterns - About - FAQs
**Description**

🛠️ The goal of this issue is to track the implementation of the `About - FAQs` pattern. 
🧑‍💻 The implementation must follow the [pattern creation guidelines](https://github.com/WordPress/twentytwentyfive/blob/trunk/README.md#pattern-creation-guidelines), and should be assigned to its appropriate category.
🎨 [Figma design](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-2076&t=IqDHO3nwfV13Hcvm-1)

About_ FAQs

### PR DESCRIPTION
**Pull request by tanvirulhaque**
**Description**

Adds Block Patterns - About FAQs #67

**Screenshots**
Screenshot 2024-09-11 at 1 38 11 PM

**Testing Instructions**

1.  Create a page.
2. Add the pattern.
3. Test that the pattern looks like the design.

### CODE REVIEW

File: patterns/text-faqs.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * Viewport width: 1400
+ * Description: A FAQs section with a FAQ heading and list of toggle questions and answers.
+ *
+ * @package    WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since      Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"metadata":{"name":"<?php echo esc_html_x( 'FAQs', 'Name of the FAQs pattern', 'twentytwentyfive' ); ?>"},"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
```
Comment (juanfra) [inline patterns/text-faqs.php]: Please remove the metadata.

File: patterns/text-faqs.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	<!-- wp:heading {"align":"wide"} -->
+	<h2 class="wp-block-heading alignwide"><?php echo esc_html_x( 'Frequently Asked Questions', 'Heading of the FAQs', 'twentytwentyfive' ); ?></h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"var:preset|spacing|30"}},"layout":{"type":"default"}} -->
+	<div class="wp-block-group alignwide">
+		<!-- wp:details {"style":{"border":{"bottom":{"color":"var:preset|color|opacity-20","width":"1px"}},"spacing":{"padding":{"bottom":"var:preset|spacing|30"}},"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
+		<details class="wp-block-details" style="border-bottom-color:var(--wp--preset--color--opacity-20);border-bottom-width:1px;padding-bottom:var(--wp--preset--spacing--30);font-style:normal;font-weight:400">
+			<summary><?php echo esc_html_x( 'What is The Stories Book about?', 'Question in the FAQs pattern', 'twentytwentyfive' ); ?></summary>
+			<!-- wp:paragraph {"placeholder":"Type / to add a hidden block","style":{"typography":{"fontStyle":"normal","fontWeight":"300"}}} -->
```
Comment (juanfra) [inline patterns/text-faqs.php]: Please remove the placeholder for the paragraph. And please do it with the other paragraphs having this.

```suggestion
			<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"300"}}} -->
```

**Review by juanfra — COMMENTED** [review]
Thank you very much for working on this! The implementation looks well. I left a few comments for you to address. 

The other thing I'd like to ask you is if you can wrap the pattern in a group that has full-width. That way users, will be able to select the variation for the pattern and it will look in line with the other patterns (taking full width). Please ensure that you set the margin to zero for that group. Thanks

https://github.com/user-attachments/assets/e2a268bf-ecd0-4fd9-b20b-ffb90060b166

File: patterns/text-faqs.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Title: FAQs
+ * Slug: twentytwentyfive/text-faqs
+ * Categories: text, about, featured
+ * Keywords: faq, about, frequently asked
```
Comment (juanfra) [inline patterns/text-faqs.php]:
```suggestion
 * Keywords: faq, about, frequently asked questions
```

**Comment by tanvirulhaque** [comment]
Hi @juanfra thanks for your quick review :) I have updated with the fix, could you take a look?

**Review by juanfra — APPROVED** [review]
Thank you so much for working on this @tanvirulhaque 

The code is looking good to me, and it is also looking good ✨

https://github.com/user-attachments/assets/6694e05e-0d48-4f8a-a940-e623eb973437

**Pull request merged**